### ISSUE TITLE
Loss is NaN or inf. Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 1.0

### ISSUE THREAD

**Issue by harrypotter90** [NVIDIA/NeMo#1225]
I am using Nemo latest tag version (v0.11.1), Training it with Swedish language, so just change the vocabulary using config file : quartznet15x5.yaml

After training for roughly 4-5 hours i start to get this error/warning:
```
[NeMo W 2020-09-26 17:52:50 actions:1479] Loss is NaN or inf.
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 1.0
[NeMo W 2020-09-26 17:52:50 actions:1479] Loss is NaN or inf.
Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 1.0
```

**Steps to reproduce:**
Checkout the code, and checkout tag version (v0.11.1).  

Used following command to train:
python speech2text.py      --asr_model=quartznet15x5.yaml   --amp_opt_level=O2  --eval_datasets=<>    --batch_size=42       --train_dataset=<>

It run for sometime, like for hours without any much improvement in the Loss/Acc

**Expected behavior**
At least Loss should decrease gradually. 

**Environment overview (please complete the following information)**

 - Environment location: PPC64le
 - Method of NeMo install: pip install

**Environment details**

If NVIDIA docker image is used you don't need to specify these.
Otherwise, please provide:
- OS version : PPC64le
- PyTorch version : 1.2.0
- Python version : 3.6

**Comment by borisgin** [comment]
The reason for NaN are "--amp_opt_level=O2". Can you try "--amp_opt_level=O1", please?
- What is the GPU which you use? 
-  batch size/ GPU and number of  GPU-s?

**Comment by harrypotter90** [comment]
Actually i use O1 amp_opt_level. I changed the size batch size to suggested (let's see).
I am using V100.

**Comment by borisgin** [comment]
The learning rate depends on the global batch size. How many GPU-s do you use?

**Comment by harrypotter90** [comment]
Agree, 2 GPUs but i can extend it to 4, if required.

**Comment by harrypotter90** [comment]
Is there a support of Gradient clipping (range) so that it does not overflow.

**Comment by borisgin** [comment]
The learning rate (LR) in most of scripts are for DGX with 8-GPU config. Can you rerun with smaller LR/ 4 ,  please?

**Comment by okuchaiev** [comment]
I would recommend you try increasing warmup_ratio or warmup_steps.

**Comment by harrypotter90** [comment]
Sure, i try it. Thank you for the suggestions. 
I alternatively used Transfer learning and it is working great.
